src/encoded/static/components/search.js

```diff
@@ -306,8 +306,8 @@ class BiosampleComponent extends React.Component {
                     <div className="result-item__data">
                         <a href={result['@id']} className="result-item__link">
                             {`${result.biosample_ontology.term_name}`}
-                            {result.biosample_ontology && result.biosample_ontology.classification === 'organoid' ? <span>{` ${result.biosample_ontology.classification}`}</span> : null}
-                            <em>{` (${result.organism.scientific_name}`}</em>
+                            {result.biosample_ontology && result.biosample_ontology.classification === 'organoid' ? <span>{` ${result.biosample_ontology.classification} (`}</span> : <span>{' ('}</span>}
```

Comment:
Looked at this again, and now wondering -- is there an extra "(" right after the biosample_ontology.classification?